apps/transport/lib/db/data_conversion.ex

```diff
@@ -7,8 +7,8 @@ defmodule DB.DataConversion do
   import Ecto.Query
 
   typed_schema "data_conversion" do
-    field(:convert_from, :string)
-    field(:convert_to, :string)
+    field(:convert_from, Ecto.Enum, values: [:GTFS])
+    field(:convert_to, Ecto.Enum, values: [:GeoJSON, :NeTEx])
```

Comment:
Il y a déjà des contraintes sur le schéma mais j'ai vu qu'on avait besoin a plusieurs endroits de pouvoir itérer sur les valeurs possibles. On peut pas avoir des `Ecto.Enum` avec des `binary()`, dommage